ApplyToBecomeInternal/ApplyToBecomeInternal/Services/IDateValidationMessageProvider.cs

```diff
@@ -0,0 +1,31 @@
+using System.Collections.Generic;
+
+namespace ApplyToBecomeInternal.Services
+{
+	public interface IDateValidationMessageProvider
+	{
+		string DefaultMessage => "Enter a date in the correct format";
```

Comment:
Just a stylistic preference but should we mark these methods and properties as public if they're public.

Reply:
Hmm, I'd not realised that default impl. stuff could now have access modifiers - I'll put that in! :)